lib/motion-spec/core.rb

```diff
@@ -26,67 +26,75 @@ module MotionSpec
     'colorized' => ColorizedOutput
   }
 
-  def self.add_context(context)
-    (@contexts ||= []) << context
-  end
-
-  def self.current_context_index
-    @current_context_index ||= 0
-  end
+  class << self
+    def add_context(context)
+      (@contexts ||= []) << context
+    end
 
-  def self.current_context
-    @contexts[current_context_index]
-  end
+    def current_context_index
+      @current_context_index ||= 0
+    end
 
-  def self.run(arg = nil)
-    unless respond_to?(:handle_specification_begin)
-      extend(Outputs[ENV['output']] || DEFAULT_OUTPUT_MODULE)
+    def current_context
+      @contexts[current_context_index]
     end
 
-    @timer ||= Time.now
+    def run(arg = nil)
+      set_default_output
 
-    if Platform.android?
-      @main_activity ||= arg
+      @timer ||= Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.